test/integration/GCClassic/README.md

```diff
@@ -71,11 +71,9 @@ The integration test scripts accept the following command-line arguments
 
 `-e /path/to/env-file` specifies the file that is used to initialize the software environment.  This file will typically contain either `module load` or `spack load` commands.
 
-`-s scheduler` specifies the choice of scheduler (case-insensitive). SLURM and LSF schedulers are currently supported.  You may omit this for running interactively.
-
 `-p partition` specifies the partition for the scheduler.  You may omit this for running interactively.
 
-`-s` specifies that the SLURM scheduler will be used to run the integration test scripts.
+`-s scheduler` specifies that scheduler (`SLURM|LSF|none`) that will be used to run the integration test scripts.  Choosing `-s none` will run compilation-only tests.
```

Comment:
"the scheduler" rather than "that scheduler"

Reply:
This comment applies to the GCHP README too.

Reply:
@lizziel: This is now removed, as we no longer specify scheduler from the command line.